packages/next/src/server/stream-utils/node-web-streams-helper.test.ts

```diff
@@ -178,7 +179,28 @@ describe('node-web-stream-helpers', () => {
     })
   })
 
-  describe('createBufferedTransformStream', () => {})
+  describe('createBufferedTransformStream', () => {
+    it('should return a TransformStream that buffers input chunks', async () => {
+      const stream = createBufferedTransformStream()
+      const input = new ReadableStream({
+        start(controller) {
+          // enquque 3 chunks of data
+          controller.enqueue(new Uint8Array([97, 98, 99, 100]))
+          controller.enqueue(new Uint8Array([101, 102, 103, 104]))
+          controller.enqueue(new Uint8Array([105, 106, 107, 108]))
+          controller.close()
+        },
+      })
```

Comment:
I think a more complex test would be something like

```
const bigChunks = [
  new Uint8Array(... some 15 bytes or so)
  new Uint8Array(... some 15 bytes or so)
]

const input = new ReadableStream({
  async pull() {
    controller.enqueue(bigChunks[0].slice(0, 5))
    // wait a microtask
    await Promise.resolve()
    controller.enqueue(bigChunks[0].slice(5, 10))
    // wait a microtask
    controller.enqueue(bigChunks[0].slice(10))
    
    // Now wait a Task
    await new Promise(r => setTimeout(r, 0))
    
    controller.enqueue(bigChunks[1].slice(0, 5))
    // wait a microtask
    await Promise.resolve()
    controller.enqueue(bigChunks[1].slice(5, 10))
    // wait a microtask
    controller.enqueue(bigChunks[1].slice(10))
    
    controller.close()
  }
})
```

Then you assert the output chunks equal `bigChunks`